ark/phenotyping/cell_cluster_utils.py

```diff
@@ -12,17 +13,17 @@
 from ark.phenotyping import cluster_helpers
 
 
-def compute_cell_cluster_count_avg(cell_cluster_path, cell_som_cluster_cols,
-                                   cell_cluster_col, keep_count=False):
+def compute_cell_cluster_expr_avg(cell_cluster_path, cell_som_cluster_cols,
+                                  cell_cluster_col, keep_count=False):
     """For each cell SOM cluster, compute the average number of associated pixel SOM/meta clusters
```

Comment:
Comment needs to be updated here as well (remove reference to pixel SOM/meta clusters)